### ISSUE TITLE
blacklist whitelist weirdness

### ISSUE THREAD

**Issue by pvinis** [rt2zz/redux-persist#224]
say i have modules `a`, `b`, `c`. i whitelist `a`, blacklist `b` and do nothing for `c`, like so `whilelist: ['a'], blacklist: ['b']`.
what happens to `c` then?

**Comment by kevmannn** [comment]
Aren't `whitelist` and `blacklist` mutually exclusive?

**Comment by pvinis** [comment]
so should I use only one or only the other, then?
ok that makes sense. ignite has a vague comment so I couldn't understand and I came to the source project to ask.

**Comment by rt2zz** [comment]
Yes, whitelist and blacklist are mutually exclusive, you can technically define both but whitelist always wins and adding a blacklist would be redundant. 

btw what is ignite?

**Comment by pvinis** [comment]
ok good to know. thanks.
here is ignite.
https://github.com/infinitered/ignite
